base/src/com/google/idea/blaze/base/project/AutoImportProjectOpenProcessor.java

```diff
@@ -67,7 +67,7 @@ public class AutoImportProjectOpenProcessor extends ProjectOpenProcessor {
   public @NotNull
   @Nls
   String getName() {
-    return Blaze.defaultBuildSystemName() + " Project";
+    return "Bazel (OG)";
```

Comment:
OG is kind of internal terminology, let's consider if there is a more clear name. On the other hand, none of "native", "Google", "old" are clear names either :/